xcode/Subconscious/Shared/Components/Detail/MemoViewerDetailView.swift

```diff
@@ -354,7 +416,67 @@ struct MemoViewerDetailModel: ModelProtocol {
     ) -> Update<Self> {
         var model = state
         model.dom = dom
-        return Update(state: model)
+        return update(
+            state: model,
+            action: .fetchTranscludePreviews,
+            environment: environment
+        )
+    }
+    
+    static func fetchTranscludePreviews(
+        state: MemoViewerDetailModel,
+        environment: MemoViewerDetailModel.Environment
+    ) -> Update<MemoViewerDetailModel> {
+        
+        guard let owner = state.owner else {
+            return Update(state: state)
+        }
+        
+        let links = state.dom.slashlinks
+            .map { value in value.toSlashlink() }
+            .compactMap { value in value }
```

Comment:
Nit: this could be one call to `compactMap`. The return value of `.toSlashlink()` is optional, and a `.compactMap` would filter out the nils.